scripts/bug_retriever.py

```diff
@@ -59,9 +59,16 @@ def retrieve_bugs(self):
         bugzilla.download_bugs(timespan_ids + labelled_bug_ids)
 
         # Try to re-download inconsistent bugs, up to three times.
+        previously_inconsistent_bugs = []
         for i in range(3):
-            bugs = bugzilla.get_bugs()
-            bug_ids = set(bug_snapshot.get_inconsistencies(bugs))
+            if i == 0:
+                bugs = bugzilla.get_bugs()
+                previously_inconsistent_bugs = bug_snapshot.get_inconsistencies(bugs)
```

Comment:
This line is the same between the two branches, you can remove it from the two branches and call it after the if-else.